examples/decode_multithreaded.rs

```diff
@@ -0,0 +1,34 @@
+use float_eq::assert_float_eq;
+use std::sync::Arc;
+use web_audio_api::context::{BaseAudioContext, OfflineAudioContext};
+
+pub fn main() {
```

Comment:
I will add a better example than this, using a threadpool. If you spawn a thread per audio file things may run out of hand